addon/mixins/plot-area.js

```diff
@@ -39,7 +39,13 @@ export default Ember.Mixin.create({
         height: height - margin.top - margin.bottom,
         width: width - margin.left - margin.right,
         outerWidth: width,
-        outerHeight: height
+        outerHeight: height,
+        margin: {
```

Comment:
No, as those are all the modified values, in the case of bottom and right. Top and left have been carried just to maintain consistency.

It's currently not used anywhere, but available for convenience in cases where it makes a difference.